sale_margin_update/tests/test_recalculate_price_margin.py

```diff
@@ -0,0 +1,75 @@
+from odoo.exceptions import UserError
+from odoo.tests.common import TransactionCase
+
+
+class TestRecalculatePriceMargin(TransactionCase):
```

Comment:
You can use this instead: https://github.com/odoo/odoo/blob/16.0/odoo/addons/base/tests/common.py#L19

for performances reasons.